static-assets/components/cstudio-common/common-api.js

```diff
@@ -8165,25 +8167,31 @@ var nodeOpen = false,
 
           $container.show();
 
+          clickHandler = function (e) {
+            if (e.target !== this)
+              return;
+
+            $container.remove();
+            $(document).unbind('click', clickHandler);
+          }
+
+          escHandler = function (e) {
+            if(e.keyCode === 27){
+              $container.remove();
+              $(document).unbind('keyup', escHandler);
+            }
+          }
+
           // Close - on button click
           $container.one('click', '.close', function(){
               $container.remove();
```

Comment:
Not unbinding `escHandler` on this path.

Define a `destroy()` and call it in all destruction paths